agent/engine/docker_client.go

```diff
@@ -77,7 +76,11 @@ const (
 	removeImageTimeout      = 3 * time.Minute
 	// ListPluginsTimeout is the timout for ListPlugins API.
 	ListPluginsTimeout      = 1 * time.Minute
-
+	// CreateVolumeTimeout is the timout for CreateVolume API.
+	CreateVolumeTimeout      = 2 * time.Minute
```

Comment:
This might be fine for `local` volumes. For non-local ones, we might require longer timeouts. How about setting this to `5m`? On that note, it'd be good add a follow-up task to make this timeout configurable as well.